### DISCUSSION TITLE
Opening a new tab in a container?

### DISCUSSION THREAD

**Discussion by pixiekat** [mbnuqw/sidebery#1851] [General]
Hi all - 

I love Sidebery but I was wondering if it was possible to hold down the plus icon (open new tab) to open a new container rather than just a new tab? Similar to the functionality in the build in vertical tabs in Nightly? Is it possible at all in a Sidebery setting?

Thanks!

**Comment by ChilledMoth** [comment]
No, but you can add additional 'new tab' buttons (shortcuts) besides the regular one for opening a new tab in an existing container on a per panel basis by configuring a tab panel.

**Reply by pixiekat** [reply]
Oooh, thanks! I didn't know about this. This is great!
